From: Tony Luck <tony.luck@intel.com>
Date: Wed, 30 Jul 2025 08:04:37 -0700
Subject: x86/cpu: Add new Intel CPU model numbers for Wildcatlake and Novalake
Git-commit: 49f848788a4d157bb6648a57963cb060fed3d56e
Patch-mainline: v6.17-rc1
References: jsc#PED-14187

Wildcatlake is a mobile CPU. Novalake has both desktop and mobile
versions.

  [ bp: Merge into a single patch. ]

Signed-off-by: Tony Luck <tony.luck@intel.com>
Signed-off-by: Borislav Petkov (AMD) <bp@alien8.de>
Link: https://lore.kernel.org/20250730150437.4701-1-tony.luck@intel.com

Acked-by: Nikolay Borisov <nik.borisov@suse.com>
---
 arch/x86/include/asm/intel-family.h |    5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/x86/include/asm/intel-family.h
+++ b/arch/x86/include/asm/intel-family.h
@@ -140,6 +140,11 @@
 
 #define INTEL_PANTHERLAKE_L		IFM(6, 0xCC)
 
+#define INTEL_WILDCATLAKE_L		IFM(6, 0xD5)
+
+#define INTEL_NOVALAKE			IFM(18, 0x01)
+#define INTEL_NOVALAKE_L		IFM(18, 0x03)
+
 /* "Small Core" Processors (Atom/E-Core) */
 
 #define INTEL_ATOM_BONNELL		IFM(6, 0x1C) /* Diamondville, Pineview */
